Robot_Adapter/Convert/ToRobot/Properties/MemberType.cs

```diff
@@ -23,15 +23,15 @@
 using RobotOM;
 using BH.oM.Adapters.Robot;
 
-namespace BH.Engine.Robot
+namespace BH.Adapter.Robot
 {
     public static partial class Convert
     {
         /***************************************************/
         /****           Public Methods                  ****/
         /***************************************************/
 
-        public static void RobotMemberType(IRobotLabel robotMemberType, FramingElementDesignProperties framingElementDesignProperties)
+        public static void ToRobot(IRobotLabel robotMemberType, FramingElementDesignProperties framingElementDesignProperties)
```

Comment:
Method not used. Can we remove? or saving for future usage?

Reply:
Would prefer to keep for now, took a bit of effort to get those params from memory.

Reply:
ok, sure, lets keep!